rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -1244,13 +1302,19 @@ public ResponseEntity<EntityModel<ProjectReleaseRelationship>> patchProjectRelea
             @Parameter(description = "Release ID.")
             @PathVariable("releaseId") String releaseId,
             @Parameter(description = "Map of release id to usage.")
-            @RequestBody ProjectReleaseRelationship requestBodyProjectReleaseRelationship
+            @RequestBody ProjectReleaseRelationship requestBodyProjectReleaseRelationship,
+            @Parameter(description = "Comment message.")
+            @RequestParam(value = "comment", required = false) String comment
     ) throws TException {
         final User sw360User = restControllerHelper.getSw360UserFromAuthentication();
         final Project sw360Project = projectService.getProjectForUserById(id, sw360User);
         Map<String, ProjectReleaseRelationship> releaseIdToUsage = sw360Project.getReleaseIdToUsage();
         ProjectReleaseRelationship updatedProjectReleaseRelationship = projectService
                 .updateProjectReleaseRelationship(releaseIdToUsage, requestBodyProjectReleaseRelationship, releaseId);
+        if (!restControllerHelper.isWriteActionAllowed(sw360Project, sw360User) && comment == null) {
+            return new ResponseEntity(RESPONSE_BODY_FOR_MODERATION_REQUEST_WITH_COMMIT, HttpStatus.ACCEPTED);
```

Comment:
In case of missing request parameter, shouldn't the response status be `BAD_REQUEST`?